src/blog/idx-malloy/index.malloynb

```diff
@@ -0,0 +1,47 @@
+>>>markdown
+# Using Google's Project IDX with Malloy
+Google recently announced [Poject IDX](https://developers.googleblog.com/2023/08/introducing-project-idx-experiment-to-improve-full-stack-multiplatform-app-development.html) and Cloud based AI powered coding development enviornment.  It turns out that this environment makes a pretty fabulous development environment for working with [Malloy](https://www.malloydata.dev).  
+
+A Project IDX development environment consists configurable clouds instance, VSCode and AI to help you.  We've configured some of our Malloy repositories so that when you open them in Project IDX, everything is setup to run, all you need to do is log into gcloud at the command line.

```

Comment:
```suggestion
A Project IDX development environment consists of a configurable cloud instance, VSCode and AI to help you.  We've configured some of our Malloy repositories so that when you open them in Project IDX, everything is setup to run, all you need to do is log into gcloud at the command line.

```